community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -502,11 +503,13 @@ def _allocate_nodes_to_placements(nodes: List[str], excl_job_id:Optional[int], l
     if not (nodeset.enable_placement and valid_placement_node(model)):
         return no_pp
     
+    max_count = calculate_chunk_size(nodeset.placement_max_distance, nodeset)
```

Comment:
`calculate_chunk_size(nodeset)`, let `calculate_chunk_size` to get `placement_max_distance` from `nodeset`

Reply:
Updated to `calculate_chunk_size(nodeset, lkp)` since the mock lkp was needed for the test in `testResume.py`